src/sandbox.rkt

```diff
@@ -199,20 +199,13 @@
 
   ;; optionally compute error/cost for input expression
   (define target-alt-data
-    (cond
-      [(test-output test)
-        (define target-train-errs-list '())
-        (define target-test-errs-list '())
+    ;; When in platform, evaluate error
+    (for/list ([(expr is-valid?) (in-dict (test-output test))] #:when is-valid?)
+      (define target-expr (fpcore->prog expr ctx))
+      (define target-train-errs (errors target-expr train-pcontext ctx))
+      (define target-test-errs (errors target-expr test-pcontext* ctx))
```

Comment:
In theory this is a little slower than it needs to be but I don't think that's an issue, this code is run once per benchmark.

Reply:
Oh—`batch-errors` is faster than calling `errors` on each alt individually, because they probably share subexpressions. But it's not worth it here, it would probably lead to uglier code.